sdk/nodejs/src/cloudClient.ts

```diff
@@ -65,13 +86,15 @@ export class DVCCloudClient {
                     })
                 }
                 return new DVCVariable({
-                    ...variable,
+                    ...variableResponse,
                     defaultValue
                 })
             })
-            .catch((err: AxiosError) => {
+            .catch((err: unknown) => {
```

Comment:
can these be `Error` instead of `unknown`?

Reply:
no, because any "catch" statement in JS can potentially catch "anything". Code can throw whatever it wants, it doesn't have to be an Error. Declaring that what it's going to catch is definitely an "Error" isn't really correct